articles/healthcare-apis/azure-api-for-fhir/fhir-paas-portal-quickstart.md

```diff
@@ -43,6 +43,9 @@ Select an existing resource group or create a new one, choose a name for the acc
 
 Confirm creation and await FHIR API deployment.
 
+> [!NOTE]
+> When creating an Azure FHIR service, you can choose between FHIR versions R4 and STU 3. R4 is the latest version, offering broader support and more features, while STU 3 is an older version still used by some systems. Once selected, the FHIR version cannot be changed, so choose the one that best aligns with your interoperability needs.
```

Comment:
Please update to state - "During creation of FHIR instance, choose one of the supported FHIR versions - R4 or STU3. To understand more about FHIR versions, visit http://hl7.org/fhir/directory.html